Data/axisAlignedResample.cmd

```diff
@@ -0,0 +1,28 @@
+C:\Dev\HASI-ex\RelWithDebInfo\resampleAxisAligned.exe C:\Dev\HASI\Data\901-L.nrrd       C:\Dev\HASI\Tibias\901-L-label.nrrd       C:\Dev\HASI\Tibias\901-L-landmarks.tfm       C:\Dev\HASI\Tibias\901-L-AA.nrrd       C:\Dev\HASI\Tibias\901-L-AA-label.nrrd
+C:\Dev\HASI-ex\RelWithDebInfo\resampleAxisAligned.exe C:\Dev\HASI\Data\901-R.nrrd       C:\Dev\HASI\Tibias\901-R-label.nrrd       C:\Dev\HASI\Tibias\901-R-landmarks.tfm       C:\Dev\HASI\Tibias\901-R-AA.nrrd       C:\Dev\HASI\Tibias\901-R-AA-label.nrrd
+C:\Dev\HASI-ex\RelWithDebInfo\resampleAxisAligned.exe C:\Dev\HASI\Data\902-L.nrrd       C:\Dev\HASI\Tibias\902-L-label.nrrd       C:\Dev\HASI\Tibias\902-L-landmarks.tfm       C:\Dev\HASI\Tibias\902-L-AA.nrrd       C:\Dev\HASI\Tibias\902-L-AA-label.nrrd
+C:\Dev\HASI-ex\RelWithDebInfo\resampleAxisAligned.exe C:\Dev\HASI\Data\902-R.nrrd       C:\Dev\HASI\Tibias\902-R-label.nrrd       C:\Dev\HASI\Tibias\902-R-landmarks.tfm       C:\Dev\HASI\Tibias\902-R-AA.nrrd       C:\Dev\HASI\Tibias\902-R-AA-label.nrrd
```

Comment:
These should be removed, all processing should be Python scripts calling Python functions in the `hasi` package.